GitCommands/Settings/AppSettings.cs

```diff
@@ -117,6 +119,41 @@ static AppSettings()
             MigrateAvatarSettings();
         }
 
+        /// <summary>
+        /// Gets the base part of the documentation link for the current application version,
+        /// which looks something like "https://git-extensions-documentation.readthedocs.org/en/latest/"
+        /// for the master branch, and "https://git-extensions-documentation.readthedocs.org/en/release-X.Y/"
+        /// for a release/X.Y branch.
+        ///
+        /// TODO: We currently use only EN language, but should maybe consider using the user's preferred language.
+        /// </summary>
+        public static string DocumentationBaseUrl
+        {
+            get => _documentationBaseUrl ?? throw new InvalidOperationException($"Call {nameof(SetDocumentationBaseUrl)} first to set the documentation base URL.");
+        }
+
+        internal static void SetDocumentationBaseUrl(string currentGitBranch)
+        {
+            if (_documentationBaseUrl is not null)
+            {
+                throw new InvalidOperationException("Documentation base URL can only be set once");
+            }
+
+            string? docVersion = "en/latest/";
```

Comment:
(not null)
Suggest we use master to point out that this is dev version of the doc (where the version could be updated)
We have used 'latest' for the released version so far

```suggestion
            string docVersion = "en/master/";
```

Reply:
Given the controversies around the "master", let's have it "en/dev"

Reply:
Then we chanhange the name of the "master" branch too.
I prefer "main" though, just as GH changed (and some longtime ClearCase usage).